clang/docs/ReleaseNotes.rst

```diff
@@ -318,6 +318,7 @@ C23 Feature Support
 ^^^^^^^^^^^^^^^^^^^
 
 - Clang now supports `N3029 <https://www.open-std.org/jtc1/sc22/wg14/www/docs/n3029.htm>`_ Improved Normal Enumerations.
+- Clang now supports `N3030 <https://www.open-std.org/jtc1/sc22/wg14/www/docs/n3030.htm>`_ Enhancements to Enumerations.
```

Comment:
I think I intentionally didn't add the Release note because aside from diagnostic message changes there were no changes to the compiler. But it still makes sense for user awareness.
We can add the context, like in other similar patches
```suggestion
- Clang now officially supports `N3030 <https://www.open-std.org/jtc1/sc22/wg14/www/docs/n3030.htm>`_ Enhancements to Enumerations. Clang already supported it as an extension, so there were no changes to compiler behavior.
```